docs/RUNNING_LOCALLY_DOCKER.md

```diff
@@ -139,5 +139,22 @@ Most people just start the stack normally, but run web outside of the standard f
 
 Details [here](https://github.com/docker/compose/issues/4677)
 
+#### bash shell
+
```

Comment:
@mbeacom i donno if adding these items to the documentation on your PR is worth it or not.. upt ot you